projects/media-viewer/src/lib/viewers/image-viewer/image-viewer.component.scss

```diff
@@ -1,67 +0,0 @@
-#viewer-wrapper {
```

Comment:
should this file not have been moved or removed?

Reply:
it should things have changed